worlds/generic/docs/setup_en.md

```diff
@@ -136,6 +136,25 @@ are rolling locally, ensure this file is edited to your liking **before** rollin
 when running the Archipelago Installation software. If you have changed settings in this file, and would like to retain 
 them, you may rename the file to `options.yaml`. 
 
+### Adding new worlds
+
+If you are generating locally, you can play with worlds that are not included in the Archipelago installation. 
+The worlds are packaged as `.apworld` files. To add a world to your installation, click the "Install APWorld" button in
+the launcher and select the `.apworld` file you wish to install. Alternatively, you can drag the `.apworld` file onto the
+launcher, manually add the `.apworld` file to the `custom_worlds` folder of your Archipelago Installation, or double-click 
+the file itself. After adding a new world, it will function like the worlds that are already packaged with 
+Archipelago.
+
+We strongly recommend that you make sure that the source of the `.apworld` is safe and trustworthy before playing with a 
+custom world.
```

Comment:
I would suggest a bit more clarity here that you should make sure it's trustworthy because it can run code on your pc, the same as any executable file would.